app/assets/stylesheets/styles/ui/inputs/inputs.scss

```diff
@@ -16,6 +16,14 @@ textarea {
   border-radius: $global-radius;
 }
 
-input.ng-invalid {
+input-invalid-notify {
```

Comment:
Are you adding a directive called input-invalid-notify? If so, is there a new directive file that should be part of this commit?